src/clj/game/cards/hardware.clj

```diff
@@ -912,7 +931,11 @@
                                  (update! (assoc (get-card state card) :Omnidrive-prog (:cid target))))}]
     :events {:card-moved {:req (req (= (:cid target) (:Omnidrive-prog (get-card state card))))
                           :effect (effect (update! (dissoc card :Omnidrive-prog))
-                                          (use-mu (:memoryunits target)))}}}
+                                          (use-mu (:memoryunits target)))}}
+    :interactions {:pay-credits {:req (req (and (= :ability (:source-type eid))
+                                                (program? target)
+                                                (= (:title (:host target)) "Omni-drive")))
```

Comment:
This could be something like `(same-card? card (:host target))`, cuz `card` in this case will be the Omni-drive, right?

Reply:
I think so. I will test this.